docs/site/Using-strong-error-handler.md

```diff
@@ -0,0 +1,251 @@
+---
+title: 'Using strong-error-handler'
+lang: en
+keywords: LoopBack 4, error handling
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Using-strong-error-handler.html
+summary:
+  Strong-error-handler is an error handler for use in both development and
+  production environments.
+---
+
+In LoopBack 4, the request handling process starts with the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html), a simple class with
+five injected helper methods in the constructor. It is the gatekeeper of all
+requests to the app. Errors are handled by one of the Sequence actions,
+[`reject`](Sequence.md#handling-errors), which calls `strong-error-handler`
+package.
+
+# strong-error-handler
+
+This package is an error handler for use in both development (debug) and
+production environments.
+
+## Supported versions
```

Comment:
just a suggestion.. it seems a bit strange to have this supported version table where there's only one entry. If only 3.x works with LB4, perhaps we can mention in the paragraph above that LB4 requires `strong-error-handler` 3.x and up? (though there's no "up" yet).

Reply:
Sounds good! I agree that it's weird to have a table that only have one item :sweat_smile: Will reword it.